From 9ef71d0c24efcb91477e9269df5a05c5729dccf2 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:29:58 -0400
Subject: [eisa] Revert: Log device resources in dmesg

Message-id: <20140719042958.19438.23095.stgit@gir.stowe>
Patchwork-id: 86031
O-Subject: [RHEL7.1 PATCH 69/81] Revert "EISA: Log device resources in dmesg"
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: bea1b0b38439ea40b1a933ff8bc0b3c506958dd6

commit bea1b0b38439ea40b1a933ff8bc0b3c506958dd6
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 17 14:57:01 2014 -0700

    Revert "EISA: Log device resources in dmesg"

    This reverts commit a2080d0c561c546d73cb8b296d4b7ca414e6860b.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/eisa/eisa-bus.c b/drivers/eisa/eisa-bus.c
index 8842cde..1b86fe0 100644
--- a/drivers/eisa/eisa-bus.c
+++ b/drivers/eisa/eisa-bus.c
@@ -288,7 +288,6 @@ static int __init eisa_request_resources(struct eisa_root_device *root,
    edev->res[i].flags = IORESOURCE_IO | IORESOURCE_BUSY;
   }
 
-  dev_printk(KERN_DEBUG, &edev->dev, "%pR\n", &edev->res[i]);
   if (request_resource(root->res, &edev->res[i]))
    goto failed;
  }
-- 
1.7.1